omnibus.rb

```diff
@@ -28,6 +28,9 @@
 # ------------------------------
 # use_git_caching false
 
+# For now, override windows builds to x86
+windows_arch :x86
```

Comment:
We might want to expose an environment variable also to make this easy to set in a dev environment or CI:

```ruby
windows_arch ENV['OMNIBUS_WINDOWS_ARCH'] || :x86
```